modules/configuring-a-pipeline-server.adoc

```diff
@@ -55,8 +55,31 @@ If you specify incorrect data connection settings, you cannot update these setti
 +
 [NOTE]
 ====
-The external database connection is secured by default. To specify an external database connection that is not secured with TLS, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource as shown here:
+The external database connection is secured by default. To specify an external database connection that is not secured with TLS, after you create the Pipeline Server, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource definition (CRD) :
```

Comment:
I think I see a space before the colon

 custom resource definition (CRD) :

Reply:
Addressed.